.clang-format

```diff
@@ -1,2 +1,17 @@
 BasedOnStyle: LLVM
```

Comment:
Why we have clang-format related changes in this PR?

Reply:
- Actually I had a merge commit in my branch owing to which when I rebased the branch, this file are also shown as changing, however I also don't know why has this occured.
- I think I might had been an error on my side while working, but actually I am searching how to resolve it.
- Should I remove that locally and then commit so that unnecessary files aren't changed??

Reply:
Ohk , sry but I will take care of this next time!!